.github/workflows/build.yml

```
@@ -0,0 +1,55 @@
+name: Create image
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+      - master
+      - ops-7908
```

```suggestion
      - OPS-*
```

Why not future proof this instead of only building this one ops-related branch?